README.md

```diff
@@ -1,4 +1,52 @@
-- [Supervisor Services Catalog](#supervisor-services-catalog)
+# Supervisor Services
+
+ Supervisor Services offered to support modern applications through vSphere Services. New services will be added over time with the goal to continue to empower your DevOps communities.
+
+## Install Supervisor Services
+
+> 📢 Services are moved to support.broadcom.com
+
+#### Please refer to [How to find and install Supervisor Services](#how-to-find-and-install-supervisor-services) to find and install supervisor services.
+
+Prior to vSphere 8 Update 1, the Supervisor Services are only available with Supervisor Clusters enabled using VMware NSX-T. With vSphere 8 U1, Supervisor Services are also supported when using the vSphere Distributed Switch (VDS) networking stack.
+
+| Supervisor Service | vSphere 7 | vSphere 8 |
+| :--: | :--: | :--: |
+| vSphere Kubernetes Service | ❌ * | ✅ <br> _requires vSphere 8.0 Update 3 or later_|
+| Consumption Interface | ❌ | ✅ <br> _requires vSphere 8.0 Update 3 or later_|
+| vSAN Data Persistence Platform Services - MinIO | ✅ | ✅ |
+| Backup \& Recovery Service - Velero | ✅ | ✅ |
+| Certificate Management Service - cert-manager | ❌ | ✅ |
+| Cloud Native Registry Service - Harbor | ❌ * | ✅ |
+| Kubernetes Ingress Controller Service - Contour | ❌ | ✅ |
+| External DNS Service - ExternalDNS | ❌ | ✅ |
+| NSX Management Proxy | ❌ | ✅ <br> _requires vSphere 8.0 Update 3 or later with Supervisor Clusters enabled using VMware NSX-T_  |
+| Data Services Manager Consumption Operator | ❌ | ✅ <br> _requires vSphere 8.0 Update 3 or later with additional configuration.<br> Please contact Global Support Services (GSS) for the additional configuration_ |
+*\* The embedded Harbor Registry and vSphere Kubernetes Service features are still available and supported on vSphere 7 and onwards.*
+
+
+## How to find and install Supervisor Services
+
+1. Log in to support.broadcom.com.
+
+2. Select My Downloads on the left hand side navigation.
+
+3. Select Free Software Downloads available HERE if you are looking for free services. Search vSphere Supervisor Services.
```

Comment:
Search vSphere Supervisor Services -> Search "vSphere Supervisor Services".

Here and other places too: use quotes to indicate what to search